fabric_service_tokens.go

```diff
@@ -0,0 +1,41 @@
+package packngo
+
+import "path"
+
+type FabricServiceTokenType string
+
+const (
+	fabricServiceTokenBasePath                        = "/fabric-service-tokens"
+	FabricServiceTokenASide    FabricServiceTokenType = "a_side"
+	FabricServiceTokenZSide    FabricServiceTokenType = "z_side"
+)
+
+// FabricServiceToken represents an Equinix Metal metro
+type FabricServiceToken struct {
+	*Href            `json:",inline"`
+	ExpiresAt        Timestamp              `json:"expires_at,omitempty"`
+	ID               string                 `json:"id"`
+	MaxAllowedSpeed  int                    `json:"max_allowed_speed,omitempty"`
+	ServiceTokenType FabricServiceTokenType `json:"service_token_type,omitempty"`
+	State            string                 `json:"state,omitempty"`
+}
```

Comment:
The `Role` field was removed in the last commit - the field is needed and should be added back in (it's present in the snippet I shared and the snippet you captured while testing).